tests/test_utils.py

```diff
@@ -258,6 +263,30 @@ class MockException(Exception):
         with self.assertRaises(MockException):
             utils.update_available(silent_fail=False)
 
+    @patch('shub.utils._is_deploy_successful', return_value=True)
+    def test_echo_short_log_if_deployed_true(self, mock_dep_success):
+        log_file = LogFile()
```

Comment:
You could also use a `mock.Mock` instance. Then you wouldn't have to define your own class